### PR TITLE
Lamp tests

### PR DESCRIPTION
**Pull request by RusherRG**
## Description

Updated LAMP stack tests and added the missing plotting scripts for all the tests with the instructions to run them.

 

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] Any dependent changes have been added to a pull request and/or merged in other modules (native-client, lind-glibc, safeposix-rust)

### CODE REVIEW

File: tests/lamp_stack/pgbench/README.md [hunk 1]
```diff
@@ -20,3 +20,9 @@ To run pgbench only
 # on consecutive runs
 ./run_pgbench_2.sh
 ```
+
+To plot the charts have a look at all the `gen_X_plots.py` scripts. To know how to run all these plotting scripts have a look at `plot.sh`
+```sh
+./plot.sh
```
Comment (rennergade) [inline tests/lamp_stack/pgbench/README.md:26]: I think this and the lamp/nginx readmes need a bit more explanation here. Mainly that you have to create a data directory and what the format for the json is. Maybe actually add the directory with sample jsons in them being like pgbench-data-example.json etc

**Review by rennergade — CHANGES_REQUESTED** [review]

**Comment by rennergade** [comment]
There's also a few merge conflicts that need to be resolved.

**Comment by RusherRG** [comment]
> There's also a few merge conflicts that need to be resolved.

Yeah, fixed the conflicts

File: tests/lamp_stack/pgbench/README.md (hunk 1 above)
Comment (RusherRG) [reply tests/lamp_stack/pgbench/README.md:26]: Yeah that makes sense. I have updated the README.md and also added a few sample JSONs, could you please review it again?

**Pull request merged**